### DISCUSSION TITLE
Accidently created a empty Icon

### DISCUSSION THREAD

**Discussion by Afraskai** [pixelit-project/PixelIt#384] [Q&A] [answered]
Hello! 

Thanks for this nice project! Can somebody delete my accidently empty created icon "1686" from the database? 

Thanks in advance!

**Comment by foorschtbar** [comment]
@o0shojo0o

**Comment by o0shojo0o** [comment, marked as answer]
> Hello!
> 
> Thanks for this nice project! Can somebody delete my accidently empty created icon "1686" from the database?
> 
> Thanks in advance!

Hi, is now deleted.
